.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "12:00"
+  open-pull-requests-limit: 10
+  commit-message:
+    prefix: patch
```

we might have to modify this to contain a colon after patch but I'm not sure.

```suggestion
    prefix: patch:
```

---

Ahh, you mean if the colon might mess up with YAML syntax? 
I think we can run with this and see what happens?

---

I was thinking about versionbot if it wants the colon or not. But checking the source versionbot actually has prefix: patch without a colon in their dependabot config: https://github.com/product-os/versionbot/blob/e0bbcef1d7b1b06147519b90b51e7fe044c3c1bc/.github/dependabot.yml#L9